applications/nrf5340_audio/src/modules/led_assignments.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _LED_ASSIGNMENTS_H_
```

Comment:
Does this need to be its own header file, or can it go into led.h?

Reply:
It is separate to match how the buttons are defined, but I could add it to led.h.
I do prefer it in it's separate header as it is thus clear where the LEDs are set to do what.